### ISSUE TITLE
[BUG] Keys and values are double qouted

### ISSUE THREAD

**Issue by p2media** [HugoGiraudel/SassyJSON#86]
Scss file:
```Sass
$map: (key1: val1, key2: (
	subkey21: val21,
	number: 1123,
	colorval: #123,
));

body:before {
	font-family: json-encode($map);
}
```

CSS file:
```css
body:before  {
	font-family: "{'\"key1\"': '\"val1\"', '\"key2\"': {'\"subkey21\"': '\"val21\"', '\"number\"': 1123, '\"colorval\"': '\"#123\"'}}";
}
```

Using:
```
node-sass	3.4.0-beta1	(Wrapper)	[JavaScript]
libsass  	3.3.0-beta2	(Sass Compiler)	[C/C++]
```

**Comment by esr360** [comment]
Try using the SassyJSON from this pull-request:

https://github.com/HugoGiraudel/SassyJSON/pull/84

Or more specifically, try replacing your "stylesheets/encode/helpers/_quote.scss" file with this:

https://github.com/esr360/SassyJSON/blob/dfcf250ddf9510a8bc0fa4b1ac5049b2cc7b9e45/stylesheets/encode/helpers/_quote.scss

**Comment by HugoGiraudel** [comment]
Should be fixed with #86.
